analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -55,6 +55,11 @@
                          Default: {_severity_map_file}
 """
 
+epilog_exit_status = """
+0 - Successful analysis
```

Comment:
Why the error code 1 is not here? The parse command can exit with this code multiple places:
```suggestion
0 - Successful analysis
1 - CodeChecker error
```
See: https://github.com/Ericsson/codechecker/blob/e02baa734ab58c696170c7289477b2ebee1d5b06/analyzer/codechecker_analyzer/cmd/parse.py#L651-L660

Please don't forget to update the docs below.